src/radar_status.py

```diff
@@ -1,46 +1,49 @@
 """
-Okay what is this for?
-
-read logs on control the radar when oopsies happen (daemon)
-
-gather misc system health information and log it (radar_status)
+TODO (Adam):
+    read logs on control the radar when oopsies happen (daemon)
+    gather misc system health information and log it (radar_status)
+    mini aggregator of nice clean logs before sending to reduce volume (radar_status)
+    console based viewer w/ plotext so we don't have to start 6 segments any more (radar_status)
+"""
 
-mini aggregator of nice clean logs before sending to reduce volume (radar_status)
+import time
+import subprocess
+import json
 
-console based viewer w/ plotext so we don't have to start 6 segments any more (radar_status)
 
-"""
+def execute_cmd(cmd):
+    try:
+        output = subprocess.check_output(cmd, shell=True)
+    except subprocess.CalledProcessError as err:
+        output = {'cmd_error': err.output}
+    return output.decode('utf-8')
 
 
-def statustype():
-    errors = ('EXPNEEDED', 'NOERROR', 'WARNING', 'EXITERROR')
-    return errors
+def inxi_cli():
+    """
+    General system health information:
+    """
+    cmd = f"inxi --tty --no-sudo --swap --disk --info --processes --sensors --output json --output-file print"
+    msg = execute_cmd(cmd)
+    msg = json.loads(msg)
+    log.info("system health", **msg)
 
-def errortype():
-    return {}  # TODO
+    return
 
 
-class RadarStatus():  # TODO finish the class when we determine how to log and what information
-    # from the driver that we would like to pass back to the experiment / user.
-    # Suggested information: confirmed ctrfreq's and sampling rate's from the driver
-    # third-stage sampling rate (rate of result data)
-    """Class to define transmit specifications of a certain frequency, beam, and pulse sequence.
-    
-    errors = ('EXPNEEDED', 'NOERROR', 'WARNING', 'EXITERROR')
-    
-    Probably will be phased out once administrator is working
-    """
+def main():
+    while True:
+        inxi_cli()
+        time.sleep(5)
 
-    def __init__(self):
-        """
-        A RadarStatus is only initiated on startup of radar_control so we need an experiment 
-        at this point. 
-        """
-        self.status = 'EXPNEEDED'# needs a control program.
-        self.errorcode = None
-        self.logs_for_user = []
 
+if __name__ == '__main__':
+    from utils import log_config
 
-    # def warning(self, warning_data):
-    #     self.status = 'WARNING'
-    #     self.logs_for_user.append(warning_data)
+    log = log_config.log(logfile=False, aggregator=False)
+    log.info(f"RADAR_STATUS BOOTED")
```

Comment:
One potential problem defining `log` within `__main__` is that this module (and other that use logging like this) can't be loaded by another module, as `log` wouldn't be defined. I came across this problem when trying to load `experiment_handler` into the experiment unittests.

Not a major problem, but something to consider as we start to implement unittests for all our modules.

Reply:
That's actually the main idea. I only want logs generated when the module is directly being run and  not, for example, imported and some functions used. This way the log files are just for when we run borealis.

Reply:
That makes sense, we may want to make some checks then so that there aren't undefined `log` variables when loading a Borealis module in.